eng/native/functions.cmake

```diff
@@ -220,6 +220,12 @@ endfunction(convert_to_absolute_path)
 function(preprocess_file inputFilename outputFilename)
   get_compile_definitions(PREPROCESS_DEFINITIONS)
   get_include_directories(PREPROCESS_INCLUDE_DIRECTORIES)
+  get_source_file_property(SOURCE_FILE_DEFINITIONS ${inputFilename} COMPILE_DEFINITIONS)
```

Comment:
I wonder if this was a compensation for the COMPILE_DEFINITIONS property that you have fixed based on my feedback. Or is this something else?

Reply:
The `get_source_file_property` was added to enable getting source-file-specific definitions. In particular, RC_INVOKED. The code your feedback was on was the pair to this line.